cmd/queue/main.go

```diff
@@ -201,8 +205,57 @@ func createAdminHandlers() *http.ServeMux {
 	return mux
 }
 
+func probeQueueHealthPath(port string) error {
+	url := fmt.Sprintf("http://127.0.0.1:%s%s", port, queue.RequestQueueHealthPath)
+
+	httpClient := &http.Client{
+		Transport: &http.Transport{
+			// Do not use the cached connection
+			DisableKeepAlives: true,
+		},
+		Timeout: 500 * time.Millisecond,
+	}
+
+	var lastErr error
+
+	err := wait.PollImmediate(100*time.Millisecond, time.Second, func() (bool, error) {
```

Comment:
Maybe rename this to `timeoutErr` to make the check below a bit more obvious?

Reply:
👍